spec/api-desktop-capturer-spec.js

```diff
@@ -38,51 +42,56 @@ describe('desktopCapturer', () => {
     let callCount = 0
     const callback = (error, sources) => {
       callCount++
-      assert.equal(error, null)
-      assert.notEqual(sources.length, 0)
+      expect(error).to.be.null()
+      expect(sources.length).to.not.equal(0)
       if (callCount === 2) done()
     }
 
     desktopCapturer.getSources({types: ['window', 'screen']}, callback)
     desktopCapturer.getSources({types: ['window', 'screen']}, callback)
   })
 
-  it('responds to subsequent calls of different options', (done) => {
+  it('responds to subsequent calls of different options', done => {
     let callCount = 0
     const callback = (error, sources) => {
       callCount++
-      assert.equal(error, null)
+      expect(error).to.be.null()
       if (callCount === 2) done()
     }
 
     desktopCapturer.getSources({types: ['window']}, callback)
     desktopCapturer.getSources({types: ['screen']}, callback)
   })
 
-  it('returns an empty display_id for window sources on Windows and Mac', (done) => {
+  it('returns an empty display_id for window sources on Windows and Mac', done => {
     // Linux doesn't return any window sources.
     if (process.platform !== 'win32' && process.platform !== 'darwin') {
       return done()
     }
+
     desktopCapturer.getSources({types: ['window']}, (error, sources) => {
-      assert.equal(error, null)
-      assert.notEqual(sources.length, 0)
-      sources.forEach((source) => { assert.equal(source.display_id.length, 0) })
+      expect(error).to.be.null()
+      expect(sources.length).to.not.equal(0)
+      sources.forEach((source) => {
+        expect(source.display_id.length).to.equal(0)
```

Comment:
Is `display_id` a string here? If yes, then wouldn't it be better to write something like this:
```js
for (const {display_id: displayId} of sources) {
  expect(displayId).to.be.a('string').and.be.empty()
}
```